services/ui-src/src/components/app/AppRoutes.tsx

```diff
@@ -37,7 +37,7 @@ export const AppRoutes = () => {
             <Route
               key={route.path}
               path={route.path}
-              element={<ReportPageWrapper route={route} />}
+              element={<ReportPageWrapper routeKey={route.path} />}
```

Comment:
Do we even need to pass this here? We could just use the pathname from the location. 🤷

Reply:
yes and pathname is safer because it's using the DB report, not the local one